src/ios/CDVWKInAppBrowser.m

```diff
@@ -758,6 +758,15 @@ - (void)createViews
         configuration.mediaPlaybackRequiresUserAction = _browserOptions.mediaplaybackrequiresuseraction;
     }
     
+    if (@available(iOS 13.0, *)) {
+        NSString *contentMode = [self settingForKey:@"PreferredContentMode"];
+        if ([contentMode  isEqual: @"mobile"]) {
```

Comment:
Objective-C noob here, two spaces required?, some for two lines below

Reply:
```suggestion
        if ([contentMode isEqual: @"mobile"]) {
```

Guess not